ext/intl/timezone/timezone_methods.cpp

```diff
@@ -84,7 +84,7 @@ U_CFUNC PHP_FUNCTION(intltz_from_date_time_zone)
 	tzobj = Z_PHPTIMEZONE_P(zv_timezone);
 	if (!tzobj->initialized) {
 		intl_error_set(NULL, U_ILLEGAL_ARGUMENT_ERROR,
-			"intltz_from_date_time_zone: DateTimeZone object is unconstructed",
+			"DateTimeZone object is unconstructed",
 			0);
 		RETURN_NULL();
 	}
```

Comment:
Note, this should probably an Error as uninitialized objects means something BAD happened.